jetty-servlet/src/test/java/org/eclipse/jetty/servlet/GzipHandlerTest.java

```diff
@@ -493,6 +509,35 @@ public void testGzipRequest() throws Exception
 
     }
     
+
+    @Test
+    public void testGzipFormRequest() throws Exception
```

Comment:
Probably need another test that does the same, but the server path does not have GzipHandler in place (perhaps via an excluded path on the GzipHandler config?)

Reply:
I already added a test to RequestTest for that case